hushline/forms.py

```diff
@@ -25,5 +26,22 @@ def __call__(self, form: Form, field: Field) -> None:
             raise ValidationError(self.message)
 
 
+class HexColor:
+    # HTML input color elements only give & accept 6-hexit color codes
+    hex_color_regex: re.Pattern = re.compile(r"^#[0-9a-fA-F]{6}$")
```

Comment:
Possibly want to make the `#` to `#?` to make it optional. Or the error should include the `#` so that users know what's missing

Reply:
This is designed to work with the ``<input type="color">`` HTML element. It only accepts & produces exactly 6-hexit color codes with the hashtag.

Addressed by (https://github.com/scidsg/hushline/pull/573/commits/2cf9518c8cac279d9f03b1ee1b4b9bf8e5148295)